tests81/Unit/CrudPanel/CrudPanelUpdateTest.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace Backpack\CRUD\Tests81\Unit\CrudPanel;
+
+use Backpack\CRUD\Tests\Unit\CrudPanel\BaseDBCrudPanelTest;
+use Faker\Factory;
+
+/**
+ * @covers Backpack\CRUD\app\Library\CrudPanel\Traits\Update
+ */
+class CrudPanelUpdateTest extends BaseDBCrudPanelTest
+{
+    public function testGetUpdateFieldsWithEnum()
+    {
+        if ((int) app()->version() < 9) {
+            return;
+        }
```

Comment:
```suggestion
```

PHP Enum casting is available on laravel 8 too

https://laravel.com/docs/8.x/eloquent-mutators#enum-casting

Reply:
@ziming probably not on all versions because tests were failing stating that `::from()` was not available. 
Tests were failing only on L8+PHP8.1 (this test only runs on PHP8.1+) and we test L9+8.1 and L8+8.1 and was failing on the second scenario with the error I described.

Reply:
I've just checked this, and it's only available in https://github.com/laravel/framework/releases/tag/v8.69.0

Backpack only enforces `^8.26`, so if you are on 8.27 it will not complain, but the test will fail. 

We had a reasonable amount of issues that upgrading Laravel constrain would help to fix, is this another one @tabacitu ?

Reply:
I thought tabacitu already said okay to bump in another PR comment previously.

https://github.com/Laravel-Backpack/CRUD/pull/4442#issuecomment-1157608688

I think in general it's okay to use features in newer version of laravel 8. Most laravel packages work this way too.

Reply:
Yes, I think it's perfect;y reasonable to bump our requirement to `^8.69` - it's the only way we can launch this. Let's do it 💪

Reply:
Bumped!